src/variant_detection/variant_output.cpp

```diff
@@ -45,15 +46,16 @@ void find_and_output_variants(std::map<std::string, int32_t> & references_length
                             tmp.add_info("SVTYPE", "DEL");
                             // Increment position by 1 because VCF is 1-based
                             tmp.set_pos(mate1_pos + 1);
-                            tmp.add_info("SVLEN", std::to_string(-distance + 1));
+                            tmp.add_info("SVLEN", std::to_string(sv_length));
                             // Increment end by 1 because VCF is 1-based
                             // Decrement end by 1 because deletion ends one base before mate2 begins
```

Comment:
These two comments seem to be outdated. There is no increment/decrement

Reply:
Yes, but only because they cancel each other out.

Reply:
:facepalm: yes, they cancel each other out, perfectly good comment :-)